daemons/compute/server/servers/server_default.go

```diff
@@ -231,10 +231,16 @@ func (s *defaultServer) Create(ctx context.Context, req *pb.DataMovementCreateRe
 	}
 
 	var dm *nnfv1alpha1.NnfDataMovement
-	if computeMountInfo.Type == "lustre" {
+	switch computeMountInfo.Type {
+	case "lustre":
 		dm, err = s.createNnfDataMovement(ctx, req, computeMountInfo, computeClientMount)
-	} else {
+	case "gfs2":
 		dm, err = s.createNnfNodeDataMovement(ctx, req, computeMountInfo)
+	default:
+		return &pb.DataMovementCreateResponse{
+			Status:  pb.DataMovementCreateResponse_INVALID,
+			Message: fmt.Sprintf("filesystem not supported: '%s'", computeMountInfo.Type),
```

Comment:
Other error messages have capitalized sentence structure i.e "Filesystem not..."

Reply:
I change the rest to be lowercase to adhere to https://staticcheck.io/docs/checks/#ST1005

Reply:
Are you running a linter in vscode that exposes this? It might be nice for all of us to use the same env.

Also surprised gofmt doesn't warn on this

Reply:
It's part of the vscode go plugin. The issues will show up under the Problems tab in the panel.

Reply:
This is what ultimately gets ran: https://staticcheck.io

Reply:
I use `revive` as my go linter within vscode.
If you look at Settings and search for Linter under the GO section you can see what you are using.